k8s/coordinator/development/config-release-coordinator.toml

```diff
@@ -0,0 +1,21 @@
+aggregator_url = "http://localhost:8082"
```

Comment:
This won't work, because it means that the `coordinator` will inform participants that in order to reach the `aggregator`, they need to connect to `http://localhost:8082`.

Reply:
I believe it should be `http://dev-aggregator.xain.io`, right?

Reply:
Yes, but I'm also adding the port (80 in this case) :)